core/src/main/java/org/bitcoinj/crypto/HDKeyDerivation.java

```diff
@@ -80,21 +80,11 @@ public static DeterministicKey createMasterPrivateKey(byte[] seed) throws HDDeri
     /**
      * @throws HDDerivationException if privKeyBytes is invalid (not between 0 and n inclusive).
      */
-    public static DeterministicKey createMasterPrivKeyFromBytes(byte[] privKeyBytes, byte[] chainCode)
-            throws HDDerivationException {
-        // childNumberPath is an empty list because we are creating the root key.
-        return createMasterPrivKeyFromBytes(privKeyBytes, chainCode, HDPath.M());
-    }
-
-    /**
-     * @throws HDDerivationException if privKeyBytes is invalid (not between 0 and n inclusive).
-     */
-    public static DeterministicKey createMasterPrivKeyFromBytes(byte[] privKeyBytes, byte[] chainCode,
```

Comment:
Why not keep a variant of this constructor with the path?

Reply:
Master private key always has path: 'm'

See: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki

Reply:
Indeed, that makes sense.